dotcom-rendering/src/web/layouts/LiveLayout.tsx

```diff
@@ -610,7 +611,15 @@ export const LiveLayout = ({ CAPI, NAV, format, palette }: Props) => {
 								/>
 							</GridItem>
 							<GridItem area="lastupdated">
-								<></>
+								{CAPI.blocks.length &&
+									CAPI.blocks[0].blockLastUpdated && (
+										<ArticleLastUpdated
```

Comment:
Amazing work, Ash! One thing I noticed: You might want to double check how `ArticleLastUpdated` sits in the grid for 740px breakpoint. I can see a shift happening. Example stories:

Liveblog: https://www.chromatic.com/test?appId=5dfcbf3012392c0020e7140b&id=620bc34e4528a3003a62839b
Deadblog: https://www.chromatic.com/test?appId=5dfcbf3012392c0020e7140b&id=620bc34e4528a3003a62839f

Reply:
Good catch, thanks Ionna!